pkgs/by-name/kr/kryptor/deps.nix

```diff
@@ -0,0 +1,9 @@
+{ fetchNuGet }: [
+  (fetchNuGet { pname = "Geralt"; version = "3.0.1"; hash = "sha256-jNiRhjqY4juSR68zHfkW+IKnnq2qXlXi3hXZrPyRlps="; })
+  (fetchNuGet { pname = "libsodium"; version = "1.0.19"; hash = "sha256-EXeaeLf3kpeFw5ecr/D/hZbOdSH+t518pV6HwOxc8ec="; })
```

Comment:
Doesn't libsodium include a native binary in it?
Don't we need to run patchelf on it to ensure it works properly?

Reply:
patchelf is currently run on the whole package dir after restore happens.  It's limited in the dependencies it can handle, but it might work.

Reply:
I tested and could encrypt and decrypt with it.